app/bundles/LeadBundle/Views/Note/note.html.php

```diff
@@ -62,7 +62,7 @@
                          </a>
                      <?php endif; ?>
                 </div>
-                <?php echo $text; ?>
+                <?php echo htmlspecialchars($text); ?>
```

Comment:
I don't think this will work. The notes can contain HTML as the WYSIWYG editor is present for the note text. So If I'd use some bold text then it would appear like this:
```php
echo htmlspecialchars('<b>bold text</b>');
```
```
&lt;b&gt;bold text&lt;/b&gt;
```
I don't think this change can make it to the production. And it wouldn't get to production anyway. All the PHP templates are removed and TWIG templates will be created instead.  Some bundles are already converted. So this file will not exist in Mautic 5 and all the XSS issues will be fixed once and for all 🎉